tests/interactive/test_app.py

```diff
@@ -12,7 +13,7 @@
 
 
 @pytest.mark.asyncio()
-async def test_input():
+async def tests_input():
```

Comment:
why this change?

Reply:
pytest works via string matching, so `test_` functions are matched specifically. I think we allow for `tests_` also but it threw me off, I'd recommend changing it back